geopyspark/geotrellis/rdd.py

```diff
@@ -120,6 +120,32 @@ def to_numpy_rdd(self):
         ser = self.geopysc.create_tuple_serializer(result._2(), value_type=TILE)
         return self.geopysc.create_python_rdd(result._1(), ser)
 
+    def to_tiled_rdd(self, extent=None, layout=None, crs=None, tile_size=256,
```

Comment:
Okay, I'll change it then.

Though, because we reference both Spark and RDDs throughout the library I don't think it's really possible to hide those terms from the user. In addition, having knowledge of Spark and its associated terms would allow for the user to better understand the advantages/disadvantages of using GeoPySpark vs another geospatial library.